crates/op-rbuilder/README.md

```diff
@@ -77,34 +75,32 @@ git checkout op-rbuilder
 - Windows: `{FOLDERID_RoamingAppData}/reth/`
 - macOS: `$HOME/Library/Application Support/reth/`
 
-3. Run a clean OP stack in the `optimism` repo:
-
-```bash
-make devnet-clean && make devnet-down && make devnet-up
-```
-
-4. Run `op-rbuilder` in the `rbuilder` repo on port 8547:
+3. Run `op-rbuilder` in the `rbuilder` repo on port 4444:
 
 ```bash
-cargo run -p op-rbuilder --bin op-rbuilder --features optimism -- node \
-    --chain ../optimism/.devnet/genesis-l2.json \
-    --http \
-    --http.port 8547 \
-    --authrpc.jwtsecret ../optimism/ops-bedrock/test-jwt-secret.txt \
+cargo run -p op-rbuilder --bin op-rbuilder -- node \
+    --chain $HOME/.playground/devnet/l2-genesis.json \
+    --http --http.port 2222 \
+    --authrpc.port 4444 --authrpc.jwtsecret $HOME/.playground/devnet/jwtsecret \
+    --port 30333 --disable-discovery \
     --metrics 127.0.0.1:9001 \
-    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+    --rollup.builder-secret-key ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 \
+    --trusted-peers enode://3479db4d9217fb5d7a8ed4d61ac36e120b05d36c2eefb795dc42ff2e971f251a2315f5649ea1833271e020b9adc98d5db9973c7ed92d6b2f1f2223088c3d852f@127.0.0.1:30304
 ```
 
-5. Init `mev-flood`:
+4. Init `contender`:
 
 ```bash
-docker run mevflood init -r http://host.docker.internal:8547 -s local.json
+git clone https://github.com/flashbots/contender
+cd contender
+contender setup 
```

Comment:
I guess it must be the section 5.

Reply:
Ah sorry, this is a typo. The command below with the `cargo run` is all you should need. Let me fix this.